datasets/br/bcb_prohibited/crawler.py

```diff
@@ -0,0 +1,103 @@
+from typing import List
+from zavod import Context, helpers as h
+
+
+def fetch_data(context: Context) -> List[dict]:
+    """
+    Fetches data from the website, or raises an exception on failure.
+
+    :param context: The context object.
+    :return: List of dicts containing data.
+    """
+    response = context.fetch_json(context.data_url)
+    if "value" not in response:
+        context.log.error("Value not found in JSON")
+        return []
+    return response["value"]
+
+
+def create_entity(input_dict: dict, context: Context):
+    """
+    Creates an entity from the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - CPF
+                        - Nome
+    :param context: The context object.
+    """
+    entity = context.make("LegalEntity")
+    tax_number = input_dict["CPF_CNPJ"]
+
+    # If it's 14 digits, then it is a CNPJ: https://en.wikipedia.org/wiki/CNPJ
+    if len(tax_number) == 14:
+        entity.add_schema("Company")
+
+    # If it's 11 digits, then it is a CPF: https://en.wikipedia.org/wiki/CPF_number
+    elif len(tax_number) == 11:
+        entity.add_schema("Person")
+
+    # It cannot be anything other than those two
+    else:
+        context.log.error("Tax number is neither a CPF nor CNPJ")
+
+    entity.id = context.make_id(tax_number)
+    entity.add("name", input_dict["Nome"])
+    entity.add("taxNumber", tax_number)
+    entity.add("country", "br")
+    entity.add('topics', 'corp.disqual')
```

Comment:
For this list I think debarment is most appropriate

```suggestion
    entity.add('topics', 'debarment')
```